src/index.tsx

```diff
@@ -185,6 +188,9 @@ export const Hint: React.FC<IHintProps> = props => {
         } else if (caretIsAtTextEnd && allowTabFill && e.key === TAB && hint !== '') {
             e.preventDefault();
             handleOnFill();
+        } else if (caretIsAtTextEnd && allowEnterFill && e.key === ENTER && hint !== '') {
```

Comment:
Please add unit tests to cover this `allowEnterFill` case as was done for `allowTabFill`.